src/boot_O2/gfxprint.c

```diff
@@ -34,12 +27,12 @@ void GfxPrint_InitDlist(GfxPrint* this) {
         gDPSetTileSize(this->dlist++, i * 2, 0, 0, 60, 1020);
     }
 
-    gDPSetPrimColorMod(this->dlist++, 0, 0, this->color.rgba);
+    gDPSetColor(this->dlist++, G_SETPRIMCOLOR, this->color.rgba);
```

Comment:
```suggestion
    gDPSetColor(this->dList++, G_SETPRIMCOLOR, this->color.rgba);
```
could you change all the `dlist` -> `dList` in this file? It seems it's not 100% consistent, but `dList` is currently used 10x more than `dlist` and it's an abbreviation of two words, so I think this makes sense